src/dss/utils.py

```diff
@@ -45,6 +62,21 @@ def wait_for_deployment_ready(
             logger.info(f"Deployment {deployment_name} in namespace {namespace} is ready")
             break
         elif time.time() - start_time >= timeout_seconds:
```

Comment:
I vote for increating the timeout to at least 180 seconds ... sometimes I am running tests on smaller aws instances and the mlflow deployment on initialise is timing out ... I think 180 is reasonable.

Reply:
Agreed, setting it as a default for both when creating a notebook and initializing DSS.